docs/architecture/adr/ADR032-sre-permissions.md

```diff
@@ -2,7 +2,7 @@
 
 ## Status
 
-Accepted, supplemented by [ADR038](ADR038-sre-permissions-istio.md)
+Superceded by [ADR043](ADR043-k8s-resource-access.md)
```

Comment:
```suggestion
Superseded by [ADR043](ADR043-k8s-resource-access.md)
```
Supersede is a hard word.. I don't think there's any other English word that has -sede as a suffix like this.